aliveevie/code_of_conduct.md

```diff
@@ -0,0 +1,48 @@
+# code of conduct and reporting guidelines
+# Teams & Roles
+Here are defined the primary teams participating in conda-forge activities.
+- **core:** The core team is the governing body over the entire conda-forge organization. Members of the core team have full rights over all conda-forge repositories. Members of core are the face of the project, and are responsible for officially interfacing with external communities, organizations, non-profits, and companies. The core team may create new sub-teams, as appropriate. Each member of core is entitled to one vote on all elected matters.
+
+- **staged-recipes:** The staged-recipes team administers the staged-recipes repository and is responsible for the review and creation of new feedstocks.
+
+- **maintainers:** A maintainer is an individual responsible for the management of one or more feedstock repositories and packages. Maintainers have the ability to merge pull requests into the feedstock of only the packages they maintain.
+
+- **external contributors:** This group encompasses all others who are not on core, part of staged-recipes, or maintainers. This includes first-time contributors, collaborators, and funders. They have no special rights within the conda-forge organization itself.
+
+- **emeritus-core:** Core members that are inactive (commits, GitHub comments/issues/reviews, dev meetings and voting on polls) in the past six months will be asked if they want to become emeritus-core developers. Any core member can also request to become emeritus if they wish to do so (e.g. taking a sabbatical or long vacation). Emeritus core members can still vote and move back to active core anytime. Emeritus votes are used to count towards quorum but the quorum size is computed from the size of the active core group. The core.csv list should be updated when a change in the status of a member occurs.
+

```

Comment:
Did you get the information above from anywhere in particular or did you do some research and put it together?